client/services/SolutionHandlerService.js

```diff
@@ -49,9 +49,9 @@ tie.factory('SolutionHandlerService', [
             studentCode.trim());
           CodePreprocessorDispatcherService.preprocess(
             language, codeSubmission, auxiliaryCode,
-            task.getMainFunctionName(), task.getOutputFunctionName(),
-            task.getCorrectnessTests(), task.getBuggyOutputTests(),
-            task.getPerformanceTests());
+            task.getInputFunctionName(), task.getMainFunctionName(), 
```

Comment:
I noticed some trailing spaces here and realized our linter wasn't checking that, sorry. I just sent #193 for review -- once it's in, please merge master into your branch and fix these. Thanks!